iGCS/Managers/GCSDataManager.m

```diff
@@ -20,13 +23,46 @@ + (instancetype) sharedInstance {
     return sharedInstance;
 }
 
+
 - (instancetype) init {
     self = [super init];
     if (self) {
         _craft = [GCSCraftModelGenerator createInitialModel];
         _lastViewedMapCamera = nil;
+       [self loadArchives];
     }
     return self;
 }
 
+- (void)loadArchives {
+    NSData *decodedData = [NSData dataWithContentsOfFile:[GCSDataManager filePath]];
+    GCSSettings *progData = [NSKeyedUnarchiver unarchiveObjectWithData:decodedData];
+    if (progData) {
+        _gcsSettings = progData;
+    }
+    else {
```

Comment:
Style. else statements should be on the same line `} else {`. Please fix throughout.